src/github.ts

```diff
@@ -0,0 +1,23 @@
+interface Release {
+  name: string;
+  createdAt: Date;
+  tagName: string;
+}
+
+interface ApiRelease {
+  name: string;
+  created_at: string;
+  tag_name: string;
+}
+
+function loadReleases({ owner, repo }: { owner: string; repo: string }): Promise<Release[]> {
+  return fetch(`https://api.github.com/repos/${owner}/${repo}/releases`)
```

Comment:
Interesting [the docs](https://docs.github.com/en/rest/releases/releases#list-releases) don't specify it, but I suppose it's safe to assume these are ordered by most recent creation date?

Reply:
With longer spans of time, would we have to deal with pagination?